app/src/main/java/org/pesmypetcare/mypetcare/activities/fragments/community/groups/PostsFragment.java

```diff
@@ -273,6 +272,16 @@ private void initializeButtons(CircularEntryView circularEntryView, View options
         btnReport.setOnClickListener(v -> {
             addReportButtonListener(circularEntryView, editPostDialog);
         });
+
+        User user = InfoGroupFragment.getCommunication().getUser();
+        if (selectedPost.isBanned() && forum.getOwnerUsername().equals(user.getUsername())) {
+            MaterialButton btnUnban = optionsPostLayout.findViewById(R.id.unbanPostButton);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line matches the illegal pattern 'Use ViewBinding instead'.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5643405)